src/services/utils/index.js

```diff
@@ -40,6 +40,22 @@ export const getLastPrice = (product) => {
   return 0;
 };
 
+/**
+ * Get Avarage Price.
+ */
+
+export const getAveragePrice = (prices) => {
+  let sum = 0;
+
+  for (let it = 0; it < prices.length; it++) {
```

Comment:
Would something like this be a better approach to a for loop?

const sum = prices.map(price => parseFloat(getFormattedPrice(price))).reduce((acc, val) => acc + val, 0)